api.py

```diff
@@ -166,7 +192,13 @@ def assemble_release_data(data: str):
                 "translations": int(translations["count"]),
             },
             "flavors": {
-                "stable": assemble_release_data(stable_data),
```

Comment:
Does it make sense to split this up into `"github": {"stable": {...}, ...}` and `"f-droid": {"stable": {...}, ...}` instead? GitHub names releases differently, i.e. there's releases and pre-releases.

Reply:
Sounds good.